pkg/networks/commands_darwin_test.go

```diff
@@ -0,0 +1,26 @@
+package networks
+
+import (
+	"path/filepath"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestVDESock(t *testing.T) {
+	config, err := DefaultConfig()
+	assert.NilError(t, err)
+
+	vdeSock := config.VDESock("foo")
+	varRunDir := filepath.Join("/", "private", "var", "run", "lima")
```

Comment:
No need to use filepath.Join

Reply:
It was just using the same code as copy/pasted everywhere else (maybe needs a function?) but I can convert it to a native string again (now that it is Darwin-only)